extensions/docs/msal-node-extensions.md

```diff
@@ -63,35 +133,52 @@ const linuxPersistence = LibSecretPersistence.create(cachePath, serviceName, acc
 An unencrypted file persistence, which works across all platforms, is provided for convenience, although not recommended.
 
 ```js
-import { FilePersistence } from "@azure/msal-node-extensions";
+const { FilePersistence } = require("@azure/msal-node-extensions");
+
+const filePath = "path/to/cache/file.json";
+const filePersistence = await FilePersistence.create(filePath, loggerOptions);
+// Pass the persistence to msal config's cachePlugin
+msalConfig.cache.cachePlugin = new PersistenceCachePlugin(filePersistence);
+ // Initialize the electron authenticator
+authProvider = new AuthProvider(msalConfig); 
 
-const cachePath = "path/to/cache/file.json";
-const filePersistence = FilePersistence.create(cachePath, serviceName, accountName);
 ```
 
+> :https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/e7ea9fd970c035aaee9f8d3c3d3196334fd1c0de/extensions/msal-node-extensions/src/persistence/FilePersistence.ts#L18: If file or directory has not been created, `FilePersistence.create()` will create the file and any directories in the path recursively.
+
 ### Creating the cache plugin
```

Comment:
> ### Creating the cache plugin

Can we retitle this section? We already covered how to create the cachePLuing above, the crucial piece of information in this section is the part about this:

> To support concurrent access by multiple processess, the extensions use a file based lock. You can configure the retry number and retry delay for lock acquisition through CrossPlatformLockOptions.